src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -606,13 +673,28 @@ export class MeetingManager implements AudioVideoObserver {
     );
   };
 
+  unsubscribeFromSelectedAudioInputTransformDevice = (
+    callbackToRemove: (device: Device | AudioTransformDevice | null) => void
+  ): void => {
+    this.selectedAudioInputTransformDeviceObservers = this.selectedAudioInputTransformDeviceObservers.filter(
+      callback => callback !== callbackToRemove
+    );
+  };
+
   private publishSelectedAudioInputDevice = (): void => {
     for (let i = 0; i < this.selectedAudioInputDeviceObservers.length; i += 1) {
       const callback = this.selectedAudioInputDeviceObservers[i];
       callback(this.selectedAudioInputDevice);
     }
   };
 
+  private publishSelectedAudioInputTransformDevice = (): void => {
+    for (let i = 0; i < this.selectedAudioInputTransformDeviceObservers.length; i += 1) {
+      const callback = this.selectedAudioInputTransformDeviceObservers[i];
+      callback(this.selectedAudioInputTransformDevice);
+    }
```

Comment:
```typescript
for (const observer of this.selectedAudioInputTransformDeviceObservers) {
```

and fix that for the others, too.

Reply:
Thanks for pointing this out, updated all of them.